application/rest/beneficiary.py

```diff
@@ -54,4 +55,37 @@ def create_beneficiary():
         status=STATUS_CODES[response.type],
     )
 
-    # curl   -X POST -H "Content-Type: application/json" -d '{"firstName":"sample","lastName":"lname","middleName":"mname","phone":"99872111","email":"email2@sample"}'  http://localhost:8000/api/v1/beneficiaries
\ No newline at end of file
+@blueprint.route("/api/v1/beneficiaries/search", methods=["POST"])
+def beneficiary_search():
+    search_input = request.json['search_input']
+    response = search_beneficiary(current_app.config.get('REPO'), search_input)
+    return Response(
+        json.dumps(response.value, cls=BeneficiaryJsonEncoder),
+        mimetype="application/json",
+        status=STATUS_CODES[response.type],
+    )
+
+@blueprint.route("/api/v1/beneficiaries/<id>", methods=["GET"])
+def beneficiary_view(id):
+    response = view_beneficiary(current_app.config.get('REPO'), id)
+    return Response(
+        json.dumps(response.value, cls=BeneficiaryJsonEncoder),
+        mimetype="application/json",
+        status=STATUS_CODES[response.type],
+    )
+
+@blueprint.route("/api/v1/beneficiaries/<id>", methods=["POST"])
+def beneficiary_update(id):
+    firstName = request.json['firstName']
```

Comment:
hope you are removing whitespaces around names.

Reply:
Next time add these checks and validations on server .  Its always good to be resilient programmer